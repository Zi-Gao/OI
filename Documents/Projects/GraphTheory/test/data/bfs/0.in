172 153
99 129 8976
35 59 6338
27 7 15617
69 134 18062
73 115 8897
37 127 13081
14 125 15032
131 44 15412
59 167 3953
126 104 22372
92 115 4161
164 90 31188
143 27 28451
39 103 18784
21 94 5351
115 135 28120
103 4 7226
153 155 17855
17 117 22839
62 76 1080
154 125 30339
77 69 26084
25 71 29219
29 84 18641
54 15 4995
48 89 6108
5 36 8564
159 48 13314
31 37 31234
107 1 26270
105 130 1908
43 81 32136
30 164 13527
153 112 3718
68 74 18908
133 118 4507
100 16 18678
124 96 6671
1 94 23547
10 110 1097
140 19 23222
142 80 720
125 106 21543
126 73 7953
47 139 3687
43 78 4793
22 54 31336
28 5 25795
16 153 28691
114 53 398
67 94 31843
9 48 10134
23 76 28307
128 87 2723
156 51 29316
125 105 3466
97 130 13781
80 158 30425
167 17 8814
121 57 7936
79 95 29804
130 127 22232
48 46 10919
18 136 15254
55 112 22616
145 107 27021
118 77 895
71 130 4424
167 127 20246
22 87 27366
116 95 321
122 26 29124
63 61 21739
125 125 18124
45 31 17308
165 65 8716
51 7 24428
63 102 27745
128 113 15955
141 53 82
16 110 12113
13 50 15739
134 132 18211
104 90 7183
14 108 5499
165 11 22111
3 56 21748
109 60 7269
101 122 30530
7 33 10689
123 169 23620
13 119 4278
159 25 7683
136 23 11516
19 30 29903
71 165 1925
46 16 7657
1 68 4238
157 153 13442
111 18 1932
71 130 24571
124 3 16598
161 9 27554
167 88 18205
139 17 7014
171 121 11024
55 30 11831
66 85 15805
135 53 18955
27 64 656
81 13 22207
149 58 27646
15 90 25676
129 71 573
77 109 28176
27 40 30108
71 157 28690
140 26 26193
36 22 9356
91 63 27099
46 130 12066
138 38 6734
135 124 18571
27 147 22047
42 146 30880
30 35 22339
118 124 26116
147 12 32460
129 46 2531
55 53 11840
119 113 31614
27 148 23135
153 40 23163
45 30 15127
161 104 28726
3 55 10957
17 144 16256
35 152 31572
72 148 14225
106 114 20763
138 66 27165
51 86 1406
17 28 19908
9 54 14142
55 89 3419
84 29 7310
95 111 31168
36 2 27701
18 15 14129
167 166 23471
129 80 99
17 94 20067
103 163 10444
