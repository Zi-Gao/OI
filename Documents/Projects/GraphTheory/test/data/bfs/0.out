1 68 74 94 